client/src/components/shop/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import Card from './ItemCard';
+import fakeData from './fakeData';
+
+export default class Shop extends Component {
+  state = {};
+
+  render() {
+    return (
+      <div className="items-list">
+        {fakeData &&
+          fakeData.map((item, i) => <Card key={i.toString()} item={item} />)}
```

Comment:
>We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state.

[good to read](https://reactjs.org/docs/reconciliation.html#recursing-on-children.)
If I were you, I'd use `key={item.id}` so react will always compare with the real id and will not mutate it.